workshops/3d_models_with_zdog/README.md

```diff
@@ -0,0 +1,257 @@
+
+![Animated model](https://cloud-djkuut7y4.vercel.app/0screen_recording_2020-10-03_at_7.31.47_pm.gif)
+
+Amazing, huh?
+
+![Amazing gif](https://cloud-hrs0t8jeh.vercel.app/0tenor.gif)
+
+### Multiple Shapes
+If you want to try and make more complex models you will need to use multiple shapes, [here](https://zzz.dog/#made-with-zdog) are some examples of what you can create:
+
+![examples](https://cloud-2jaw6a14x.vercel.app/0image.png)
+
+Making multiple shapes is very easy, it’s as simple as putting multiple shapes together, until you get what you want. The hardest part of this, is to put everything where it must be, we can do this using the property: “translate”. Let’s see how it works!
+
+I wanna try making the Hack Club logo, but you can make whatever you want!. Click [here](https://repl.it/@wollygfx/Hack-Club-logo) to see the final code.
+
+I'll start making the red square in the background:
+```javascript
+new Zdog.Box({
+  addTo: model,
+  width: 100,
+  height: 100,
+  color: '#ec3750',
+  stroke:20,
+  translate: { z: -18 },
+})
+```
+1. As seen before, the ```addTo``` property puts the made shape inside of the main Zdog Illustration.
+2. Used the width and height properties to make a perfect square, i didn't give it a depth because it was not needed.
+3. The whole square will have the same color (red), so we can use the color property to give the whole shape a single color instead of assigning a color to each face.
+4. As said before, the stroke property helps the shape to look a little rounded.
+5. The translate property moves the square -18 within the z-axis. You can interpret this as if you were giving a shape a depth, so the shape is moved backwards.
+
+
+When running these few lines, we will get this:
+![red square](https://cloud-2q1vsewnu.vercel.app/0image.png)
+
+That looks perfect!. Now I will create the letter h, so I am gonna need 3 more boxes:
+```javascript
+new Zdog.Box({
+  addTo: model,
+  depth:20,
+  width: 20,
+  height:80,
+  color: '#fff',
+  translate: { z: 18, x:-20},
+})
+```
+
+- This time, the shape is moved forward... This creates an space between the red square and this new shape.
+- The shape is moved to the left within the x-axis
+
+![result 1](https://cloud-kg0xtr3hs.vercel.app/0image.png)
+
+```javascript
+new Zdog.Box({
+  addTo: model,
+  depth:20,
+  width: 20,
+  height:40,
+  color: '#fff',
+  translate: { z: 18, y:20,x:20,},
+})
+```
+
+- This time, we created a new box but way smaller.
+- We moved it to the right within x-axis and a little bit down within the y-axis.
+
+![result 2](https://cloud-1nisp19i8.vercel.app/0image.png)
+
+```javascript
+new Zdog.Box({
+  addTo: model,
+  depth:20,
+  width: 40,
+  height:20,
+  color: '#fff',
+  translate: { z: 18, x:10},
+})
+```
+In this last one, all we had to do was to move the box to the right within the x-axis, so that it blends with the box in the right.
+![result 3](https://cloud-m2gpkvlqa.vercel.app/0image.png)
+
+Now, i will just animate it 
```

Comment:
```suggestion
```
In this last one, all we had to do was to move the box to the right within the x-axis, so that it blends with the box in the right.
![result 3](https://cloud-m2gpkvlqa.vercel.app/0image.png)

Now, I will just animate it 
```